### ISSUE TITLE
Windows getting 404 on download update

### ISSUE THREAD

**Issue by ryanpager** [ArekSredzki/electron-release-server#64]
So im having trouble getting the windows updates to work correctly. Ive set up an update version (1.0.1) on the server, and updated the assets.

(win32) XXX-1.0.1-full-ia32.nupkg
(win64) XXX-1.0.1-full.nupkg
(mac) XXX-1.0.1-mac.zip

When the windows build trys to go to download the update, I get the following error from squirrel,

"The remote server returned an error: (404) Not Found."

When I curl the following url,

"curl -i https://<my_server>/update/windows_64/1.0.1/RELEASES"

I get a 204 No Content (which I think is correct).

Is there some sort of naming convention im screwing up, or something wrong im doing?

Side Note -- im generating the url like this:

```
const version = package.version
const platform = os.platform() + '_' + os.arch()
switch(os.arch()) {
    case 'ia32':
      platform = 'windows_32';
    break;
    case 'x64':
      platform = 'windows_64';
    break;
  }

End Result:
https://auto-updater.servername.com/update/${platform}/${version}
```

**Comment by ryanpager** [comment]
@ArekSredzki Any ideas on this?

**Comment by ArekSredzki** [comment]
Do you have an access log? That would help you check whether a different endpoint than what's expected is being returned.

**Comment by ryanpager** [comment]
@ArekSredzki Does this help any?

```
Windows Update Search Query { platform: [ 'windows_64' ],
  version: '1.0.0',
  channel: 'stable' }
Applicable Channels [ 'stable' ]
Time Filter { '>=': Mon Aug 15 2016 15:43:45 GMT+0000 (UTC) }
Latest Windows Version { assets:
   [ { name: 'TalkFusion-1.0.1-full.nupkg',
       platform: 'windows_64',
       filetype: '.nupkg',
       hash: 'D53732F41D87FC700B6DBE7D950EC10B79B68D0D',
       size: 111118276,
       download_count: 0,
       fd: '/opt/release/fab189d5-1e81-4be2-9a27-d05e0c66cadd.nupkg',
       createdAt: '2016-08-22T19:36:37.000Z',
       updatedAt: '2016-08-22T19:36:37.000Z',
       version: '1.0.1' } ],
  channel: 'stable',
  name: '1.0.1',
  notes: '- Testing auto updater',
  createdAt: '2016-08-22T19:34:33.000Z',
  updatedAt: '2016-08-22T19:34:33.000Z' }
```

Were running v1.2.1 right now, and the url that we are hitting is,

<base_url>/update/windows_64/1.0.1

Really stumped on this...could it be a file naming thing?

**Comment by ryanpager** [comment]
@ArekSredzki Fixed by upgrading to 1.3.1 (latest) -- thanks for the help!
